lib/user_app.dart

```diff
@@ -193,10 +193,18 @@ class UserApp extends StatelessWidget {
                                     builder: (_) => QRScan(),
                                     settings: settings,
                                   );
-                                case '/withdraw_funds':
+                                case '/withdraw_funds_spontaneous':
```

Comment:
good catch! Actually spontaneous sounds a bit misleading here because it sounds like keysend and here we only want to send lightning funds  and swap them with onchain.
Maybe use:
`unexpected_funds `- for the case where we have funds in the onchain wallet
`withdraw_funds` - for the case where we want to send to btc address.

Saying all this I think we need to disable the "Send to btc address" for now as it is not implemented in the sdk yet.

Reply:
You are right; I did change the names to your suggestion, which sounds better.
I did disable the send to BTC address too.
@erdemyerebasmaz I did apply your suggestion as well